.gitignore

```
@@ -1 +1,2 @@
 node_modules
+dist
```

```suggestion
dist

```
Add new line to end of file